exercises/rotational-cipher/rotational_cipher_test.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+
+import rotational_cipher
+
+
+class RotationalCipher(unittest.TestCase):
```

Comment:
We try to stick to the [canonical test data](https://github.com/exercism/x-common/blob/master/exercises/rotational-cipher/canonical-data.json) provided in the `x-common` repository.

If you are missing a test case that would make sense for all tracks, than it would be best to also add it there.
If you think we need an additional test case that would be python specific it would be good to mark those with a comment (`# additional track specific tests`) to make it easy to compare it against the common test cases.